spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+RSpec.describe 'users#show', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Lilly', photo: 'lilly.jpg', bio: 'Software Developer', posts_counter: 0)
+    @posts = [
+      Post.create(author: @user, title: 'Post 1', text: 'This is Post 1'),
+      Post.create(author: @user, title: 'Post 2', text: 'This is Post 2'),
+      Post.create(author: @user, title: 'Post 3', text: 'This is Post 3')
+    ]
+    visit user_path(@user)
+  end
+  it 'displays the user\'s bio' do
+    expect(page).to have_content(@user.bio)
+  end
+  it 'displays the user\'s posts' do
+    @posts.each do |post|
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text)
+      expect(page).to have_content("Comments: #{post.comments.count}")
+      expect(page).to have_content("Likes: #{post.likes.count}")
+    end
+  end
+  it 'displays a "See all posts" button if the user has more than 3 posts' do
+    expect(page).to have_selector('button', text: 'See all posts')
+  end
+  it 'redirects to the user posts index page when "See all posts" button is clicked' do
+    visit user_path(@user)
+    click_link 'See all posts'
```

Comment:
Nice job so far😉. Just as [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md) in the `user show page` kindly add the following tests😉.

- I can see the user's profile picture.
- I can see the user's username.
- I can see the number of posts the user has written.
- I can see the user's first 3 posts.
- When I click a user's post, it redirects me to that post's show page.
<img width="688" alt="image" src="https://github.com/Innocent259/blog-app/assets/64643995/8eefb723-1b0f-4d20-bbec-a5f645f4eab5">